frontend/lib/Views/AdminPage/utils.dart

```diff
@@ -81,66 +81,98 @@ PreferredSize adminAppBar(String upperText, String bottomText) => PreferredSize(
-        child: Text(
-          bottomText,
-          style: const TextStyle(fontSize: 25),
+      preferredSize: const Size.fromHeight(120),
+      child: AppBar(
+        iconTheme: const IconThemeData(
+          color: adminGreenColor,
+        ),
+        centerTitle: true,
+        toolbarHeight: 120,
+        title: Container(
+          padding: const EdgeInsets.only(top: 20, bottom: 10),
+          child: const Text("Administrator panel"),
         ),
+        backgroundColor: Colors.white,
+        titleTextStyle: const TextStyle(
+          color: Colors.black,
+          fontFamily: 'Ubuntu Condensed',
+          fontSize: 40,
+          fontWeight: FontWeight.w500,
+        ),
+        titleSpacing: 20,
+        bottom: bottomText != ""
+            ? PreferredSize(
+                preferredSize: const Size.fromHeight(20),
+                child: Container(
+                  padding: const EdgeInsets.only(bottom: 10, top: 5),
+                  child: Text(
+                    bottomText,
+                    style: const TextStyle(fontSize: 25),
+                  ),
+                ),
+              )
+            : null,
       ),
-    ) : null,
-  ),
-);
+    );
 
-Container buildDeleteContainer(Function onDeletePressed, arguments) => Container(
-  alignment: Alignment.centerRight,
-  padding: const EdgeInsets.symmetric(vertical: 20, horizontal: 20),
-  child: OutlinedButton(
-    style: ButtonStyle(
-      side: MaterialStateProperty.all(
-        const BorderSide(width: 1.5, color: Colors.red),
-      ),
-      padding: MaterialStateProperty.all(
-        const EdgeInsets.all(15),
-      ),
-      foregroundColor: MaterialStateProperty.resolveWith<Color>(
+Container buildDeleteContainer(Function onDeletePressed, arguments) =>
+    Container(
+      alignment: Alignment.centerRight,
+      padding: const EdgeInsets.symmetric(vertical: 20, horizontal: 20),
+      child: OutlinedButton(
+        style: ButtonStyle(
+          side: MaterialStateProperty.all(
+            const BorderSide(width: 1.5, color: Colors.red),
+          ),
+          padding: MaterialStateProperty.all(
+            const EdgeInsets.all(15),
+          ),
+          foregroundColor: MaterialStateProperty.resolveWith<Color>(
             (Set<MaterialState> states) =>
-        states.contains(MaterialState.hovered)
-            ? Colors.white
-            : Colors.red,
+                states.contains(MaterialState.hovered)
+                    ? Colors.white
+                    : Colors.red,
+          ),
+          backgroundColor: MaterialStateProperty.resolveWith<Color>(
+            (Set<MaterialState> states) =>
+                states.contains(MaterialState.hovered)
+                    ? Colors.red
+                    : Colors.white,
+          ),
+          alignment: Alignment.centerLeft,
+        ),
+        child: const Icon(Icons.delete_outline_outlined, size: 30),
+        onPressed: () => onDeletePressed(arguments),
```

Comment:
IMO to nie jest dobry pomysł. Dużo lepszym pomysłem jest wyciągnąć `onPressed` jako callback i przekazywać tylko jego w parametrze funkcji `buildDeleteContainer`. 
```dart
Container buildDeleteContainer(Function onPressed) =>
```

Dzięki temu wywołanie odpowiedniej funkcji może nastąpić przy użyciu funkcji `buildDeleteContainer`
```dart
void onDeleteDoSomething() {
    myDeleteFunctions(myArguments)
}

buildEditContainer(onDeleteDoSomething)
```

Reply:
Dlaczego to nie jest dobry pomysł? Potrzebuję tych konkretnych argumentów. Wskaż mi proszę jak inaczej mam to zrobić.

Reply:
Imo to nie jest komentarz na ten task - było mówione o refaktorze i ujednoliceniu stylów kupę razy, więc nie ma co dodawać komentarz z tym upomnieniem, jak to już było mówione milion razy i będzie robione

Reply:
> Dlaczego to nie jest dobry pomysł? Potrzebuję tych konkretnych argumentów. Wskaż mi proszę jak inaczej mam to zrobić.

1. Nie masz sprecyzowanego typu argumentów, które przekazujesz do funkcji `onDeletePressed`, jest to typ `Any`
2. Musisz do `buildDeleteContainer` przekazać dwa argumenty, zamiast jednego
3. Co w przypadku, gdy będziesz chciał wywołać funkcję, która nie przyjmuje żadnych argumentów, co przekażesz jako `arguments`? Null?

Reply:
1. Bo o to chodzi żebym przekazał cokolwiek
2. I jaki z tym jest problem? Bo nie rozumiem?
3. Tutaj przekazuje, a nawet jak przekażę nulla -> mogę przekazać to w formie mapy (tak jak przekazujesz do wszystkich flutterowych widgetów parametry), i zrobić null-safety funkcję. Tak jak np. w Zostawiłem w tej postaci bo po raz kolejny: Refactor incomming

Tutaj problem jest taki, że przypisujemy przycisk per user -> jakoś musimy przekazać informacje o tym który to użytkownik, więc rezygnowanie z parametru wydaje się bez sensu.

Reply:
> IMO to nie jest dobry pomysł. Dużo lepszym pomysłem jest wyciągnąć `onPressed` jako callback i przekazywać tylko jego w parametrze funkcji `buildDeleteContainer`.
> 
> ```dart
> Container buildDeleteContainer(Function onPressed) =>
>    OutlinedButton(
>        onPressed: onPressed
>    )
> ```
> 
> Dzięki temu wywołanie odpowiedniej funkcji może nastąpić przy użyciu funkcji `buildDeleteContainer`
> 
> ```dart
> void onDeleteDoSomething() {
>     myDeleteFunctions(myArguments)
> }
> 
> buildEditContainer(onDeleteDoSomething)
> ```

Sugerowałbym jeszcze raz przejrzeć moje rozwiązanie, jeśli nie w tej PR'ce, to przy planowanym refaktorze